docs/Readme.md

```diff
@@ -1,13 +1,15 @@
 # Unity ML-Agents Documentation
 
+## Installation
+ * [Installation & Set-up](Installation.md)
+    * [Background: Jupyter Notebooks](Background-Jupyter.md)
+ * [Docker Set-up](Using-Docker.md)
```

Comment:
Indent

Reply:
The reason for this indent was because the [Background: Jupyter notebook] was a subpage of Installation & Set-up page, just like all of the other Background page. 

I moved the Docker Set-up page one level up because I think it should be a parallel part of Installation, but we can move its level down.